tests/tests.js

```diff
@@ -451,11 +451,25 @@ exports.defineAutoTests = function () {
 
                 it("filetransfer.spec.9 should not leave partial file due to abort", function (done) {
 
+                    jasmine.addMatchers({
+                        toBeAbortOrConnectionError: function() {
+                            return {
+                                compare: function(transferError) {
+                                    var result = {pass: transferError === FileTransferError.ABORT_ERR || transferError === FileTransferError.CONNECTION_ERR};
+                                    if(!result.pass) {
+                                        result.message = "Expected " + transferError + " to be " + FileTransferError.ABORT_ERR + " or " + FileTransferError.CONNECTION_ERR;
+                                    }
+                                    return result;
+                                }
+                            }
+                        }
+                    });
```

Comment:
@dblotsky The issue is that the result could be connection error or abort error. Both are valid in this case. So, I have created my own matcher.